File: c-sharp/ex/constructors/heritage/exercises/animal-inheritance/Cerdo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AnimalInheritance
{
  class Cerdo : Animal
  {
    int kilosCarne;
    public Cerdo(string sonido) : base(sonido) { }

    public void Comunicarse()
    {
      Console.WriteLine("Cuando el cerdo pesa mucho, hace " + sonido);
    }
  }

}

File: java/jse-workspace/fundamentos/src/com/pablomonteserin/clasesabstractasinterfaces/claseabstracta/Cerdo.java
package com.pablomonteserin.clasesabstractasinterfaces.claseabstracta;

public class Cerdo extends Animal {
	int kilosCarne;
	
	Cerdo(String sonido) {
		super(sonido);
	}

	@Override
	public void comunicarse() {
		System.out.println("El cerdo cuando pesa mucho hace " + sonido);
	}

}
